test/corectl_integration_test.go

```diff
@@ -223,7 +223,7 @@ func TestCorectl(t *testing.T) {
 		{"project 1 - get objects", defaultConnectString1, []string{"object", "ls"}, []string{"golden", "project1-objects.golden"}, initTest{true, true}},
 		{"project 1 - get object data", defaultConnectString1, []string{"object", "data", "my-hypercube"}, []string{"golden", "project1-data.golden"}, initTest{true, true}},
 		{"project 1 - get object properties", defaultConnectString1, []string{"object", "properties", "my-hypercube"}, []string{"golden", "project1-properties.golden"}, initTest{true, true}},
-		{"project 1 - get measures 1 as json", defaultConnectString1, []string{"measure", "ls"}, []string{"golden", "project1-measures-1-json.golden"}, initTest{true, true}},
+		{"project 1 - get measures 1", defaultConnectString1, []string{"measure", "ls"}, []string{"golden", "project1-measures-1.golden"}, initTest{true, true}},
```

Comment:
We should add a test for `minimum` flag as well if that is the flag name we decide upon